packages/hop-node/src/watchers/classes/Token.ts

```diff
@@ -5,34 +5,47 @@ import { formatUnits, parseUnits } from 'ethers/lib/utils'
 
 export default class Token extends ContractBase {
   tokenContract: ERC20
+  isEth: boolean
```

Comment:
minor nitpick

the condition is already a boolean so the assignment can be shortened to

```js
this.isEth = (this.tokenContract.address === constants.AddressZero)
```